docs/release.md

```diff
@@ -0,0 +1,55 @@
+# Status AddOn Relesea process 
+
+The Status AddOn release process is based on [GoReleaser](https://goreleaser.com).
+GoReleaser is configured to automatically create a new release by pushing new tags matching the
+pattern 'v*' and build and publish to ghcr.io the packages for the 
+[container image](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon)
+and the [helm chart](https://github.com/kubestellar/ocm-status-addon/pkgs/container/ocm-status-addon-chart) 
+used to deploy the status add-on. The relevant files for goreleaser configuration are
+[./goreleaser.yaml](../.goreleaser.yaml) and [./.github/workflows/goreleaser.yml](../.github/workflows/goreleaser.yml).
+
```

Comment:
The remainder of the current draft seems strange to me. It seems to be focused on git mechanics, and they seem confused to me. What is needed is high-level direction. I think the directions that need to be given are about when to create a new release branch and what commits to apply release tags (those starting with "v") to. Following is an example of what such directions might look like. Feel free to disagree and write something different, I just offer this as an illustration of the sort of things that need to be said.

The main line of development is done in the git branch named `main`. There are also release branches, with names like `release-0.1`, on which patches to existing releases can be made.

A release is identified by "major.minor.patch" numbers, according to [semantic versioning](semver.org).

Start a new release branch by making it the same as main (`git checkout main; git pull; git branch -b release-$major.$minor`). Continue work on any release branch in the usual way for extending a branch.

Create a release by applying a git tag of the form `v$major.$minor.$patch`. This should be applied to a commit in the branch named `release-$major.$minor`. That commit should also be in `main` if `$patch` is 0.